apps/src/craft/designer/CraftVisualizationColumn.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+var msg = require('@cdo/locale');
+
+var GameButtons = require('../../templates/GameButtons').default;
+var BelowVisualization = require('../../templates/BelowVisualization');
+var ArrowButtons = require('../../templates/ArrowButtons');
+import ProtectedVisualizationDiv from '../../templates/ProtectedVisualizationDiv';
```

Comment:
nit: could use more ES6 (import instead of var, const instead of var for our class, export default instead of module.exports